test/unittests/mongoBackend/mongoContextProvidersQuery_test.cpp

```diff
@@ -1442,7 +2056,7 @@ TEST(mongoContextProvidersQueryRequest, severalCprs1)
   EXPECT_EQ("E1", RES_CER(0).entityId.id);
   EXPECT_EQ("T", RES_CER(0).entityId.type);
   EXPECT_EQ("false", RES_CER(0).entityId.isPattern);
-  EXPECT_EQ("", RES_CER(0).providingApplication);
+  EXPECT_EQ(0, RES_CER(0).providingApplicationList.size());
```

Comment:
As disccussed time ago, "general" entity-level CPr is now a list instead of a single value.